apps/docs/src/guide/contracts/transaction-dependency-estimation.md

```diff
@@ -0,0 +1,11 @@
+# Transaction Dependency Estimation
+
+Previously, we mentioned that a contract call might require you to manually specify external contracts or variable outputs.
```

Comment:
Would be good to link to or reference what 'Previously' is referring to here incase this is the first page the reader is landing on.

Reply:
Good catch! @danielbate